scripts/docker-quick-start/sql/Dockerfile

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2022 Apollo Authors
```

Comment:
Move Dockerfile to other folder? folder `scripts/docker-quick-start/sql` is prepare for .sql file